Library/Formula/validns.rb

```diff
@@ -0,0 +1,30 @@
+class Validns < Formula
+  homepage "http://www.validns.net/"
+  url "http://www.validns.net/download/validns-0.8.tar.gz"
+  sha256 "df2db0eaa998a0411ff4c1c4e417eb82d32aec4835f92f45f26c66c8d1d5bd22"
+
+  depends_on 'judy'
```

Comment:
Use double-quotes everywhere (and fix everything else suggested by `brew audit --strict validns`)